modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/background/task/CTPublishBackgroundTaskExecutor.java

```diff
@@ -100,7 +108,7 @@ public BackgroundTaskResult execute(BackgroundTask backgroundTask)
 		if (!_ctSchemaVersionLocalService.isLatestCTSchemaVersion(
 				fromCTCollection.getSchemaVersionId())) {
 
-			throw new IllegalArgumentException(
+			throw new CTPublishConflictException(
```

Comment:
Don't change the exception type. This exception happens the publication is out of date, meaning there's a db schema change.